prettier.config.cjs

```diff
@@ -0,0 +1,7 @@
+/** @type {import("prettier").Config} */
```

Comment:
Why cjs? could you use typescript or javascript instead?

Reply:
> Why cjs? could you use typescript or javascript instead?

They simply don't support it. From Prettier docs:

> - A "prettier" key in your package.json file.
> - A .prettierrc file written in JSON or YAML.
> - A .prettierrc.json, .prettierrc.yml, .prettierrc.yaml, or .prettierrc.json5 file.
> - A .prettierrc.js, .prettierrc.cjs, prettier.config.js, or prettier.config.cjs file that exports an object using module.exports.
> - A .prettierrc.toml file.

cjs is currently the closest we have to ts